pkg/project/auth/cache.go

```diff
@@ -67,6 +67,35 @@ func subjectRecordKeyFn(obj interface{}) (string, error) {
 	return subjectRecord.subject, nil
 }
 
+type State interface {
+	State() string
+}
+
+type reflectorState struct {
+	reflectors []*cache.Reflector
+}
+
+func (rs *reflectorState) State() string {
+	state := ""
+	for i := range rs.reflectors {
```

Comment:
Not too worried about collisions, since all that matters is the opaque token is different than the last time reviewed.